src/libs/ReportUtils.js

```diff
@@ -451,13 +451,9 @@ function getDisplayNamesWithTooltips(participants, isMultipleParticipantReport)
  * @returns {String}
  */
 function getReportName(report, personalDetailsForParticipants = {}, policies = {}) {
-    if (lodashGet(report, 'reportNameValuePairs.type') !== 'chat') {
-        return lodashGet(report, 'reportName', '');
-    }
-
     let formattedName;
     if (isChatRoom(report)) {
-        formattedName = `#${report.reportName}`;
```

Comment:
CC: @roryabraham 
Without this change we'd be seeing double `#` in room names because we already add the `#` in [`Report.getChatReportName` in actions/Report.js](https://github.com/Expensify/App/blob/b3fa99ac9585ee699adb05739dc324f836258aaa/src/libs/actions/Report.js#L149-L169)

If instead we want to simply get rid of `Report.getChatReportName` and [return `report.reportName` in these lines instead](https://github.com/Expensify/App/blob/b3fa99ac9585ee699adb05739dc324f836258aaa/src/libs/actions/Report.js#L211-L213) and do the specific chatType name logic here in ReportUtils.js, we can do that instead. I feel like keeping both of these functions is redundant.

Reply:
Yeah, I'd love to nix `Report.getChatReportName` and consolidate into `ReportUtils.getReportName`